src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -5105,10 +5140,9 @@ module Queries = struct
     (* incentivized testnet-specific queries *)
 
     let auth =
-      field "auth" ~typ:bool
-        ~args:Arg.[]
-        ~doc:"Returns true if query is authorized"
-        ~resolve:(fun _ _ -> Some true)
+      field "auth" ~args:[] ~typ:(non_null string)
```

Comment:
It wasn't specified in spec, but I thought about it now:

let's include both uuid and sequential number for the public key used in `Authorization`.

Reply:
`auth` response now includes the fields:
- `uuid`
- `signerSequenceNumber`

Is that enough?